exercises/nth-prime/nth_prime_test.py

```diff
@@ -22,11 +22,34 @@ def test_there_is_no_zeroth_prime(self):
         with self.assertRaisesWithMessage(ValueError):
             prime(0)
 
-    # additional track specific test
+    # Additional tests for this track
+
     def test_first_twenty_primes(self):
```

Comment:
I'm not sure why this test existed to begin with; it doesn't add additional coverage. If getting the 10001st prime works (case `test_big_prime` above), this is redundant. I would vote to remove the additional test altogether.

@yawpitch thoughts?

Reply:
If I were to guess it's to keep someone from writing:

```python
def prime(n):
    if n == 1:
        return 2
    if n == 2:
        return 3
    if n == 6:
        return 13
    if n == 10001:
        return 10474
    raise ValueError("LOL, whut?")
```

... without any friction.

Arguably there probably should be a test for the first N primes, but I see no reason for that to be a track-specific choice.

Reply:
I hadn't thought of that. In that case, perhaps keep the additional test, but call it by a different property? Our current structure would allow the following, I think:
```JSON
{
  "cases": [
    {
      "description": "first twenty primes",
      "property": "(lambda n: [prime(n) for n in range(1, n + 1))",
      "input": {
        "number": 20
      },
      "expected": [2, 3, 5, 7, 11, 13, 17, 19, 23, 29, 31,
                   37, 41, 43, 47, 53, 59, 61, 67, 71]
    }
  ]
}
```

Reply:
The above would need to somehow bypass the `to_snake` filter for `property`

Reply:
Another option would be to just hard-code the below into the template, then reference `prime_range` as the property:

```Python
def prime_range(n):
"""Returns a list of the first n primes"""
    return [prime(i) for i in range(1, n + 1)]
```

Reply:
I went the route of hard coding the prime_range support function into the template and calling that through the additional test case. Doing so let me drop an additional case from the test function macro which seemed a little cleaner.